Wasted/FoodPlaceRatingSystem/RatingToEmoji.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+namespace Wasted.FoodPlaceRatingSystem
+{
+    /// <summary>
+    /// Includes functions for representing numeric values of rating in user-friendly way
+    /// </summary>
+    public class RatingToAssociation
+    {
+        /// <summary>
+        /// Converts rating numeric value to emoji. Rating should be between 1 and 5
+        /// If rating value is less than 1 it is interpreted as 1
+        /// If rating value is greater than 5 it is interpreted as 5
+        /// </summary>
+        /// <param name="rating">Value of rating</param>
+        /// <returns>Emoji representation for provided 'rating'</returns>
+        public static String ConvertToEmoji(int rating)
+        {
+            if (rating <= 1) return "🤢";
+            else if (rating == 2) return "😡";
+            else if (rating == 3) return "😐";
+            else if (rating == 4) return "😋";
+            else if (rating >= 5) return "😍";
+
```

Comment:
there is no integer value that bypasses all statements and returns ""

Reply:
Switch statement or array of emojis is preferred. Pseudocode:
Emojis = { :d, xd, :( }
return Emojis[rating - 1]